vitest.config.mts

```diff
@@ -24,7 +24,7 @@ export default defineConfig(({ mode }) => {
         },
         test: {
             testTimeout: 20_000,
-            hookTimeout: 30_000,
+            hookTimeout: 60_000,
```

Comment:
As you can see by the commits on this PR, the create pool tests I just refactored will timeout on ~half the attempts. I think because I decided to move pool creation into the `beforeAll` hook. Let me know if this is bad practice. I just though it made sense to re-use same pool when testing deployment and initialization 

https://github.com/balancer/b-sdk/blob/976817520607576ce28ec6e93726510556222020/test/v3/createPool/stable/stable.integration.test.ts#L84-L88

Reply:
These tests have always been a pain! Not sure if we're missing something but haven't had time to check yet. Lets see how it goes for now.